resources/hooks/vagrant/commit-msg

```diff
@@ -13,7 +13,7 @@ COMMIT_MSG_FILE=$1
 COMMIT_MSG=$(cat "${COMMIT_MSG_FILE}")
 
 # Fetch the GIT diff and format it as command input:
-DIFF=$(git -c diff.mnemonicprefix=false --no-pager diff -r -p -m -M --full-index --no-color --staged | cat)
+DIFF=$(git -c diff.mnemonicprefix=false --no-pager diff -r -p -m -M --full-index --no-color --staged --src-prefix="a/${HOOK_PREFIX}" --dst-prefix="b/${HOOK_PREFIX}" | cat)
```

Comment:
I'm really not sure if this file needs to be updated as well. I did just to be sure, but have no way of testing it.